### PR TITLE
Find Older Generation EC2 Instances

### PR DESCRIPTION
**Pull request by harshal-netserv**
## Description
Find Older Generation EC2 Instances

### Testing
image

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

**Review by dougsillars — DISMISSED** [review]

File: AWS/legos/aws_find_older_generation_ec2_instances/aws_find_older_generation_ec2_instances.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+##
+from pydantic import BaseModel, Field
+from typing import List
+from unskript.connectors.aws import aws_get_paginator
+import pprint
+
+
+class InputSchema(BaseModel):
+    region: str = Field(
+        title='Region',
```
Comment (shloka-bhalgat-unskript) [inline AWS/legos/aws_find_older_generation_ec2_instances/aws_find_older_generation_ec2_instances.py:13]: region should be optional

File: AWS/legos/aws_find_older_generation_ec2_instances/aws_find_older_generation_ec2_instances.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    region: Optional[str] = Field(
+        default ='',
+        title='Region',
+        description='AWS Region.')
+
+
+def aws_find_older_generation_ec2_instances_printer(output):
+    if output is None:
+        return
+    pprint.pprint({"Instances": output})
```
Comment (shloka-bhalgat-unskript) [inline AWS/legos/aws_find_older_generation_ec2_instances/aws_find_older_generation_ec2_instances.py:22]: Do we need "Instances": in the print statement?

File: AWS/legos/aws_find_older_generation_ec2_instances/1.png
Comment (shloka-bhalgat-unskript) [inline AWS/legos/aws_find_older_generation_ec2_instances/1.png:1]: Can you update the output image with the check lego format

**Review by dougsillars — DISMISSED** [review]

**Review by dougsillars — APPROVED** [review]

File: AWS/legos/aws_find_older_generation_ec2_instances/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>Find Older Generation EC2 Instances</h1>
+
+## Description
+This action finds older generation EC2 instances, which are ['t2', 'm1', 'm2', 'm3', 'm4', 'c1', c3] type.
```
Comment (amit-chandak-unskript) [inline AWS/legos/aws_find_older_generation_ec2_instances/README.md:5]: Isnt the previous generation list like [this](https://aws.amazon.com/ec2/previous-generation/)

**Comment by amit-chandak-unskript** [comment]
Curious, where did this lego requirement came from?

**Pull request closed without merging**